espnet2/train/reporter.py

```diff
@@ -559,9 +572,23 @@ def wandb_log(self, epoch: int = None, commit: bool = True):
             for key2 in self.stats[epoch][key1]:
                 if key2 in ("time", "total_count"):
                     continue
-                d[f"{key1}_{key2}_epoch"] = self.stats[epoch][key1][key2]
+                key = f"{key1}_{key2}_epoch"
+                d[wandb_get_prefix(key) + key] = self.stats[epoch][key1][key2]
         d["epoch"] = epoch
-        wandb.log(d, commit=commit)
+        wandb.log(d)
+
+    def wandb_log_image(self, wandb_image_dict):
+        wandb.log(wandb_image_dict)
```

Comment:
Thank you for contributing espnet!

`wandb_log_image` and `wandb_log_model_artifact` are not need to  be methods since they doesn't refer any fields of reporter.
Instead, it's okay to invoke wandb.log and wandb.log_artifact in trainer code directly.

Reply:
Sure. I'll make these changes